tests/cupy_tests/sparse_tests/test_construct.py

```diff
@@ -96,16 +97,19 @@ def test_random_with_state(self):
         self.assertEqual(x.format, self.format)
 
         state2 = cupy.random.RandomState(1)
-        y = cupy.sparse.random(
+        y = getattr(cupy.sparse, self.random_method)(
             3, 4, density=0.1,
             format=self.format, dtype=self.dtype,
             random_state=state2)
 
         self.assertTrue((x.toarray() == y.toarray()).all())
 
     def test_random_with_data_rvs(self):
+        if self.random_method == 'rand':
+            # cupy.sparse.rand does not support data_rvs
+            return
```

Comment:
How about using `pytest.skip('cupy.sparse.rand does not support data_rvs')` here instead of just putting `return`?

Reply:
oh, thank you